src/components/widget-player.jsx

```diff
@@ -249,26 +249,20 @@ const WidgetPlayer = ({instanceId, playId, minHeight='', minWidth='',showFooter=
 				if (isPreview) {
 					args.push(inst.id)
 				}
-				const newQueue = [{ request: args }]
+				var newQueue = [{ request: args }]
```

Comment:
Does this assignment have to happen, or could we save the line (and the use of `var`) by just doing this as we call `_pushPendingLogs`?

Not worth holding anything up over, but it would be cleaner.

Reply:
I agree, more of a holdover from how the component was originally written. Updated.